Library/Homebrew/tab.rb

```diff
@@ -407,4 +430,151 @@ def to_s
     end
     s.join(" ")
   end
+
+  sig { returns(Hash) }
+  def to_spdx_sbom
```

Comment:
Is this an officially defined format somewhere? If so: we should be using some sort of validator here ideally both at `brew bottle` and `brew tests` times.

Reply:
I believe the SPDX spec is defined here: https://spdx.github.io/spdx-spec/v2.3/

In terms of a Ruby validator, I'm not sure. I can look into this 🙂

Reply:
full ack. get your results validated. the current implementation produces invalid data.
see https://github.com/Homebrew/brew/pull/16594#issuecomment-2016609230